integration/test_ml.py

```diff
@@ -133,17 +139,20 @@ def check_model(model, args):
     assert model.etag is not None
 
 
-def check_model_format(model, has_model_format=False, validation_error=None):
+def check_model_format(model, has_model_format=False, validation_error=None, is_automl=False):
```

Comment:
I'd advise having 2 methods:

```
check_tflite_model(model, has_format=False, validation_error=None)
check_automl_model(model, has_format=False)
```

Reply:
I split it up, but a bit differently - automl is also TFLite format just with a different source, so that might be misleading.
Also, tflite would have contained both models with a tflite_gcs format and models with no format.
Since we will likely implement more than TFLite models, I split it into 3: models with no format, models with tflite format from gcs source, and models with tflite format from automl source. We can then add additional format/source combinations easily in future.